src/solvers/refinement/string_refinement.cpp

```diff
@@ -923,7 +923,66 @@ void string_refinementt::substitute_array_access(exprt &expr) const
   }
 }
 
-/// negates the constraint and add it to the solver. the intended usage is to
+/// Negates the constraint and adds it to the solver. The intended usage is to
+/// find an assignement of the universal variable that would violate the axiom,
+/// to avoid false positives the symbols other than the universal variable
+/// should have been replaced by there valuation in the current model
+/// \param axiom: the not_contains constraint to add the negation of
+/// \param val: the existential witness for the axiom
+/// \param solver: the solver
+/// \return: univ_var: the valuation of the universal variable for the negation
+///   of the axiom.
+void string_refinementt::add_negation_of_not_contains_constraint_to_solver(
+  const string_not_contains_constraintt &axiom,
```

Comment:
No worries. It's just that things such as:
```java
not_contains.exists_lower_bound();
```
make more sense to me than:
```java
axiom.exists_lower_bound();
```
But that's a detail. You can leave as it is, especially if it's done in that way elsewhere.